src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -90,6 +176,24 @@ const ApplicationLauncher = _.flow(
       setupWelder()
       setShouldSetupWelder(true)
     }
+
+    const findOutdatedAnalyses = withErrorReporting('Error loading outdated analyses', async () => {
+      const outdatedRAnalyses = await checkForOutdatedAnalyses({ googleProject, bucketName })
+      setOutdatedAnalyses(outdatedRAnalyses)
+      if (outdatedRAnalyses?.length >= 1) {
+        setFileOutdatedOpen(true)
+      }
```

Comment:
```suggestion
      _.isEmpty(outdatedRAnalyses) && setFileOutdatedOpen(true)
```
We typically use the `<predicate> && doSomething()` pattern for simple conditionals like this, for concision.

Reply:
ah yes i remember seeing this pattern, will update